src/main/resources/bidder-config/pgamssp.yaml

```diff
@@ -1,6 +1,7 @@
 adapters:
   pgamssp:
     endpoint: http://us-east.pgammedia.com/pserver
+    gvlVendorID: 1353
```

Comment:
PBS Java uses different syntax. Should be `vendor-id`. It's already present in the yaml file, just set to `0` right now.

Reply:
Good catch. I've made the adjustment to `vendor-id` and removed `gvlVendorID`